src/features/content-answers/messages.ts

```diff
@@ -32,6 +32,11 @@ const messages = defineMessages({
             'Disabled tooltip message for Content Answers entry point button when the file type is not supported',
         id: 'boxui.contentAnswers.disabledTooltipFileNotCompatible',
     },
+    inlineErrorText: {
+        defaultMessage: 'The Box AI service was unavailable.',
```

Comment:
Should it say, `is` instead of `was`?

Reply:
When the error comes out, there would be a retry button for user to click. And I think it would make sense saying that Box AI service was unavailable and it is worth to retry.